Automattic/wp-calypso
# Importer: Refactor pre-migration module

Related to https://github.com/Automattic/wp-calypso/issues/82758

## Proposed Changes

* Removed obsolete/non-used components to simplify the WP migration module
* Renamed and moved component to the pre-migration folder
* Created a hook to unload the parent pre-migration component

## Testing Instructions

* Go to `/setup/import-focused/importerWordpress?siteSlug={SITE_SLUG}&from={JN_SITE}&option=everything
* Check the entering credentials flow
* Check if the flow works without problems

NOTE: For easier review, follow the commits

Generally, the module structure is now much cleaner with better component naming.

<img width="276" alt="Screenshot 2023-10-17 at 00 33 04" src="https://github.com/Automattic/wp-calypso/assets/1241413/1bc4a63a-748a-457a-a60a-a0140faccf47">


## Pre-merge Checklist
